lsp-clients.el

```diff
@@ -237,8 +237,10 @@ directory containing the package. Example:
 
 (lsp-register-client
  (make-lsp-client :new-connection (lsp-stdio-connection (lambda ()
-                                                          (cons (lsp-package-path 'typescript-language-server)
-                                                                lsp-clients-typescript-server-args)))
+                                                          `(,(lsp-package-path 'typescript-language-server)
+                                                            "--tsserver-path"
+                                                            ,(lsp-package-path 'typescript)
+                                                            ,@lsp-clients-typescript-server-args)))
```

Comment:
Do we need to delete line 250?

Reply:
Maybe, I'm not sure, I would just keep both of them.